<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGraphicsView" name="CameraView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>321</width>
     <height>381</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="SaveButton">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>360</y>
     <width>106</width>
     <height>36</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QPushButton" name="QuitButton">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>360</y>
     <width>106</width>
     <height>36</height>
    </rect>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="LCDNum">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>410</y>
     <width>321</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>10</y>
     <width>231</width>
     <height>291</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="DetailsLayout">
    <item>
     <layout class="QHBoxLayout" name="TemperatureLayout">
      <item>
       <widget class="QLabel" name="TenpertureLabel">
        <property name="text">
         <string>Temperture</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextBrowser" name="TempertureBrowser"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="Covid19Layout">
      <item>
       <widget class="QLabel" name="Covid19Label">
        <property name="text">
         <string>Covid1Result</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextBrowser" name="CovidBrower"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="NameLayout">
      <item>
       <widget class="QLabel" name="NameLabel">
        <property name="text">
         <string>Name</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="NameEdit"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="ClassNumberLayout">
      <item>
       <widget class="QLabel" name="ClassNumberLabel">
        <property name="text">
         <string>ClassNumber</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="ClassNumberEdit"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QGraphicsView" name="FaceView">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>10</y>
     <width>201</width>
     <height>291</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
